app/src/main/java/com/main/mealplanner/service/RecipeService.kt

```diff
@@ -35,7 +35,7 @@ class RecipeService {
              * exception occurred creating the request or processing the response.
              */
             override fun onFailure(call: Call<RecipeList>, t: Throwable) {
-                Log.d("TAG", t.message.toString())
+                Log.d("Error, ", t.message.toString())
```

Comment:
Look at it from point of view new developer I did not know what TAG is so I change it to "Error", to make more sense.

Reply:
Use the constant variable TAG instead.  That comes with Log.

Reply:
Ok, I didn't know that , Ill have look into it more on my end since we planning on doing this in ours later.